ocaml/xapi/xapi_globs.ml

```diff
@@ -1795,6 +1794,7 @@ module Resources = struct
       , yum_config_manager_cmd
       , "Path to yum-config-manager command"
       )
+    ; ("c_rehash", c_rehash, "Path to Regenerate CA store")
```

Comment:
`regenerate`

Reply:
As you can see, this line is moved from other place, 
Thanks for pointing this out, I will fix.